src/back/routes/events.ts

```diff
@@ -0,0 +1,25 @@
+import { WithAuth, auth } from 'decentraland-gatsby/dist/entities/Auth/middleware'
+import handleAPI from 'decentraland-gatsby/dist/entities/Route/handle'
+import routes from 'decentraland-gatsby/dist/entities/Route/routes'
+
+import { EventsService } from '../services/events'
+import { validateProposalId, validateRequiredString } from '../utils/validations'
+
+export default routes((route) => {
+  const withAuth = auth()
+  route.get('/events', handleAPI(getLatestEvents))
+  route.post('/events/voted', withAuth, handleAPI(voted))
+})
+
+async function getLatestEvents() {
+  return await EventsService.getLatest()
+}
+
+async function voted(req: WithAuth) {
+  const user = req.auth!
+
+  validateProposalId(req.body.proposalId)
+  validateRequiredString('proposalTitle', req.body.proposalTitle)
+  validateRequiredString('choice', req.body.choice)
```

Comment:
I wonder if we shouldn't use the validation schema we use for proposals here as well?

Reply:
Let's use zod for validation non-proposals schemas after merging the financial report pr.